the-collab-lab/tcl-59-smart-shopping-list
# Style App with TailwindCSS

_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._

## Description

This feature adds a look and feel to the app. It uses tailwind CSS to style the homepage, the list view, the add-item view and creates a modal to display each item when selected. Styled by Mike, Sandra, Karen, and Cassandra.
## Related Issue

 "closes #13 " 

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
| ✓   | :sparkles: New feature     |
|     | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |

## Updates


### After
![image](https://github.com/the-collab-lab/tcl-59-smart-shopping-list/assets/81008493/20c1553a-a15c-46c4-8b92-45f8208e78d0)

![image](https://github.com/the-collab-lab/tcl-59-smart-shopping-list/assets/81008493/ce43ec79-fa9f-40a6-9b90-fc9a09f7b7ff)
![image](https://github.com/the-collab-lab/tcl-59-smart-shopping-list/assets/81008493/f72b220a-e0e8-4c0e-a599-0348cc54323e)





## Testing Steps / QA Criteria

From your terminal, pull down this branch with  ` git pull origin new-feature ` and check that branch out with `git checkout 
 new-feature`
Then` npm ci `to install the newly added dependencies locally.
run `npm start` to view App in your browser
